lib/kaname/resource.rb

```diff
@@ -5,17 +5,16 @@ class Resource
     class << self
       def yaml(filename = 'keystone.yml')
         if File.exists?(filename)
-          @_yaml = YAML.load_file(filename)
-          expand_all_tenants
+          @_yaml ||= expand_all_tenants(YAML.load_file(filename))
         end
       end
 
       private
 
-      def expand_all_tenants
+      def expand_all_tenants _yaml
```

Comment:
What kind of policy does the underscore use?
I think that a commonly used example is the name of a private method etc.